[2023-06-28T09:30:27.629+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509
[2023-06-28T09:30:27.630+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice'
[2023-06-28T09:30:27.698+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509
[2023-06-28T09:30:27.699+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice'
[2023-06-28T09:30:27.751+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-28T09:28:42.663751+00:00 [queued]>
[2023-06-28T09:30:27.771+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-28T09:28:42.663751+00:00 [queued]>
[2023-06-28T09:30:27.772+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-28T09:30:27.798+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): get_choice> on 2023-06-28 09:28:42.663751+00:00
[2023-06-28T09:30:27.807+0000] {standard_task_runner.py:57} INFO - Started process 1392 to run task
[2023-06-28T09:30:27.813+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'get_choice', 'manual__2023-06-28T09:28:42.663751+00:00', '--job-id', '446', '--raw', '--subdir', 'DAGS_FOLDER/test_loop.py', '--cfg-path', '/tmp/tmpopgfnm2d']
[2023-06-28T09:30:27.815+0000] {standard_task_runner.py:85} INFO - Job 446: Subtask get_choice
[2023-06-28T09:30:27.894+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509
[2023-06-28T09:30:27.895+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-28T09:28:42.663751+00:00/task_id=get_choice'
[2023-06-28T09:30:27.898+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-28T09:28:42.663751+00:00 [running]> on host a3cd9226e3f8
[2023-06-28T09:30:28.093+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='get_choice' AIRFLOW_CTX_EXECUTION_DATE='2023-06-28T09:28:42.663751+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-28T09:28:42.663751+00:00'
[2023-06-28T09:30:28.096+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-28T09:30:28.098+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation get_stream_value --args \'{                            "stream_name": "ed_cdc_stream",                             "dry_run": false}\'']
[2023-06-28T09:30:28.113+0000] {subprocess.py:86} INFO - Output:
[2023-06-28T09:30:32.688+0000] {subprocess.py:93} INFO - [0m09:30:32  Running with dbt=1.5.1
[2023-06-28T09:30:36.473+0000] {subprocess.py:93} INFO - [0m09:30:36  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-28T09:30:36.474+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-28T09:30:36.475+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-28T09:30:36.531+0000] {subprocess.py:93} INFO - [0m09:30:36  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 457 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-28T09:30:38.785+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-28T09:30:38.832+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=get_choice, execution_date=20230628T092842, start_date=20230628T093027, end_date=20230628T093038
[2023-06-28T09:30:38.863+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-28T09:30:38.909+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
